en/appendices/4-0-upgrade-guide.rst

```diff
@@ -17,15 +17,14 @@ Once your application is running on latest CakePHP 3.x, enable deprecation warni
 
 Now that you can see all the warnings, make sure these are fixed before proceding with the upgrade.
 
-Upgrade to PHP 7.2
+Upgrade to PHP 7.3
 ==================
 
-If you are not running on **PHP 7.2 or higher**, you will need to upgrade PHP before updating CakePHP.
+If you are not running on **PHP 7.3 or higher**, you will need to upgrade PHP before updating CakePHP.
```

Comment:
I don't think this should change. We can mention that the upgrade tool requires a higher version of PHP though in the callout like you currently have.